### DISCUSSION TITLE
Inter font for Chinese characters

### DISCUSSION THREAD

**Discussion by jane-mao** [AvaloniaUI/Avalonia#14321] [Q&A] [answered]
As I know, inter font does not support Chinese, however Chinese characters can be displayed in Avalonia app. With Avalonia dev tool, I can see the font family is "compositefont:fonts:inter#inter, $Default#inter, $Default", so what's the real font family for Chinese charaters?

**Comment by kyuranger** [comment]
You'd better embed your own font inside Avalonia. I suggest using SourceHanSans or any other open-source font.

**Reply by jane-mao** [reply]
I am using inter font, and I am asking why Chinese characters can be displayed correctly with inter font in Avalonia app. I think it uses a composite font family which uses other font to display Chinese code range instead of inter font, correct? How can I get that font for Chinese code range? The dev tool just told me "compositefont:fonts:inter#inter, $Default#inter, $Default".

Thanks.

**Reply by Gillibald** [reply, marked as answer]
`FontManager.Current.TryMatchCharacter`
